services/web/server/src/simcore_service_webserver/login/decorators.py

```diff
@@ -39,7 +40,8 @@ async def wrapped(*args, **kwargs):
         if userid is None:
             raise web.HTTPUnauthorized
 
-        request[RQT_USERID_KEY] = userid
+        request[RQT_USERID_KEY] = userid[0]
+        request[RQT_USEREMAIL_KEY] = userid[1]
```

Comment:
since you return a tuple, you can do this:
```
request[RQT_USERID_KEY], request[RQT_USEREMAIL_KEY] = await authorized_userid(request)
```

Reply:
NOIS :ok_hand:

Reply:
but wait, what if it returns ``None``?

Reply:
nevermind. done